Homework5/RoutersTests/RoutersTests.cs

```diff
@@ -0,0 +1,81 @@
+using NUnit.Framework;
+using Routers;
+using System.IO;
+using System;
+
+namespace RoutersTests;
+public class RoutersTests
```

Comment:
```suggestion
namespace RoutersTests;

public class RoutersTests
```
А ещё лучше namespace в самом верху файла писать